packages/gatsby/src/utils/webpack.config.js

```diff
@@ -212,25 +212,9 @@ module.exports = async (
               compiler.hooks.done.tapAsync(
                 `gatsby-webpack-stats-extractor`,
                 (stats, done) => {
-                  let assets = {}
-
-                  for (let chunkGroup of stats.compilation.chunkGroups) {
-                    if (chunkGroup.name) {
-                      let files = []
-                      for (let chunk of chunkGroup.chunks) {
-                        files.push(...chunk.files)
-                      }
-                      assets[chunkGroup.name] = files.filter(
-                        f => f.slice(-4) !== `.map`
-                      )
-                    }
-                  }
-
                   fs.writeFile(
                     path.join(`public`, `webpack.stats.json`),
-                    JSON.stringify({
-                      assetsByChunkName: assets,
-                    }),
+                    JSON.stringify(stats.toJson({ all: false, assets: true, chunkGroups: true })),
```

Comment:
@dtinth I just check webpack/stats.js and there's an option `all` so I could reduce webpack.stats.json size without need to use custom code. maybe can I do this instead ? or It's still better to have smaller file sizes ?